shell.nix

```diff
@@ -5,16 +5,11 @@ with pkgs;
 stdenv.mkDerivation {
   name = "disciplina-nixops";
   nativeBuildInputs = [
-    awscli
+    jq
```

Comment:
If a script needs `jq`, it should specify that with its nix shell shebang, e.g. `#!nix-shell -i bash -p bash jq`. Those dependencies shouldn't be specified here.

Reply:
I removed the `nix-shell` shebang from the script, because it is extremely confusing to both start `nix-shell` manually and then have it shebanged. I’m not even sure it is a good idea in terms of how well this is supported.

Reply:
I can assure you nested `nix-shell`s work just fine. I really think a script should state its own dependencies as much as possible, rather than leaving them to the environment. In an environment where we can assume `nix` is installed, I don't see a reason not to use this to specify packages.